mlrun/projects/project.py

```diff
@@ -500,6 +500,7 @@ def setup(project):
     :returns: project object
     """
     context = context or "./"
+    project_name = _add_username_to_project_name_if_needed(name, user_project)
```

Comment:
The project name is resolved in `load_project` which is done the same as you did here but now this code is coupled with `load_project` internal logic. If someone changes `load_project` they will need to change this as well. It's true that this logic is unlikely to change but I don't see a reason to duplicate this code. You can use `project.metadata.name` where you have the project OR add the `user_project` flag to the log kwargs where you don't.

Reply:
fixed